CHANGELOG.md

```diff
@@ -8,6 +8,35 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [0.4.0] - 2023-03-15
+
+[Full changelog](https://github.com/openfga/openfga/compare/v0.3.7...v0.4.0)
+
+### Removed
+
+* [BREAKING] feat!: disable schema 1.0 support, except if appropriate flags are set (openfga/openfga#613)
+  * As of this release, OpenFGA no longer allows writing or evaluating schema `v1.0` models by default, if you need support for it for now, you can use the:
+    * `OPENFGA_ALLOW_WRITING_1_0_MODELS`: set to `true` to allow `WriteAuthorizationModel` to accept schema `v1.0` models.
+    * `OPENFGA_ALLOW_EVALUATING_1_0_MODELS`: set to `true` to allow `Check`, `Expand`, `ListObjects`, `Write` and `WriteAssertions` that target schema `v1.0` models.
+    * `ReadAuthorizationModel`, `ReadAuthorizationModels` and `ReadAssertions` are unaffected and will continue to work regardless of the target model schema version.
+  * Note that these flags will be removed and support fully dropped in a future release. Read the [Schema v1.0 Deprecation Timeline](https://openfga.dev/docs/modeling/migrating/migrating-schema-1-1#deprecation-timeline) for more details.
+
+### Added
+* feat: add OpenFGA version command to the CLI (openfga/openfga#625)
+
+### Fixed
+
+* feat: improve the speed of Check for 1.1 models by using type restrictions (openfga/openfga#545)
+* feat: tuning of OTEL parameters (openfga/openfga#570)
+* Various important fixes to the experimental ListObjects endpoint
+  * fix(list-objects): improve readUsersets query by dropping unnecessary sorting (openfga/openfga#631)
+  * fix(list-objects): fix null pointer if computed userset does not exist (openfga/openfga#572)
+  * fix(list-objects): race condition in memory store (openfga/openfga#585)
+  * fix(list-objects): ensure no objects returned that would not been allowed in Checks (openfga/openfga#577)
+  * fix(list-objects): read userset tuples with a filter like group#member (openfga/openfga#596)
```

Comment:
don't add this as a separate row, please mention the PR in line 29

Reply:
Can you elaborate? I don't follow both parts of the comment.

What do you mean about not putting 36 as a separate row?
And what do you mean by mentioning the PR in line 29? There's already a PR mentioned there

Reply:
sorry i wasn't clear, i meant that #596 was done because it was missed when we pushed #545. But people don't need to know this, IMO. They are both part of one feature